.github/actions/e2e/install-karpenter/action.yaml

```diff
@@ -83,7 +83,10 @@ runs:
         --set "serviceMonitor.endpointConfig.relabelings[2].replacement=$(git describe --abbrev=0 --tags)" \
         --set "serviceMonitor.endpointConfig.relabelings[3].targetLabel=commitsAfterTag" \
         --set "serviceMonitor.endpointConfig.relabelings[3].replacement=\"$(git describe --tags | cut -d '-' -f 2)\"" \
-        --wait
```

Comment:
Was this flag meant to be dropped? I'm guessing no since the previous line still has the trailing '\'.